firefox-ios/Shared/Strings.swift

```diff
@@ -1040,6 +1040,16 @@ extension String {
                 tableName: "FirefoxHomepage",
                 value: "Top Stories",
                 comment: "This is the title of the Top Stories section on Firefox Homepage.")
+            public static let PopularTodaySectionTitle = MZLocalizedString(
+                key: "FirefoxHome.Stories.PopularTodaySectionTitle.v144",
+                tableName: "FirefoxHomepage",
+                value: "Popular today",
+                comment: "This is the title of the Popular Today stories section on Firefox Homepage.")
```

Comment:
Same, I would explain briefly what are "Popular Today" stories in a couple of words.